raiden-dapp/src/store/version-information/getters.ts

```diff
@@ -26,21 +28,33 @@ function compareSemanticVersions(
   );
 }
 
+// This detection mechanism here isn't very nice. It is not possible to access
+// the data from the root store as it does not allow to distinguish the state.
+// Only when the user has accepted the disclaimer in the past, this data
+// exists. This also means this check here only works when evaluated before the
+// user accepts the disclaimer for the first time. In general it would be better
+// to have any other mechanism to detect the same information. Unfortunately
+// this must work with old versions of the client. This is the only and final
+// idea we came up with.
+function userLoadedApplicationForTheFirstTime(): boolean {
+  return localStorage.getItem(DISCLAIMER_STORAGE_KEY) === null;
```

Comment:
Maybe just check:
```suggestion
  return !window.localStorage.getItem(DISCLAIMER_STORAGE_KEY);
```
In case some implementation returns `undefined`? And also may be useful to access through `window` global? Just suggestions

Reply:
According to the official W3C specification it will be always `null`:
https://html.spec.whatwg.org/multipage/webstorage.html#storage

Reply:
We use plain `localStorage` over `window.localStorage` at some other locations too. I'm very sorry, but at this late point I'm not motivated to change it only at this location just to then re-trigger the CI. I hope that is fine. Sorry.

Reply:
Yes, that's fine. Let's get this merged.